src/library/scala/collection/immutable/Vector.scala

```diff
@@ -117,38 +117,38 @@ final class Vector[+A] private[immutable] (private[collection] val startIndex: I
   // requires structure is at pos oldIndex = xor ^ index
   private final def getElem(index: Int, xor: Int): A = {
     if (xor < (1 << 5)) { // level = 0
-      (display0
-      (index & 31).asInstanceOf[A])
+      display0
+        (index & 31).asInstanceOf[A]
     } else if (xor < (1 << 10)) { // level = 1
-      (display1
-      ((index >>> 5) & 31)
-        (index & 31).asInstanceOf[A])
+      display1
+        ((index >>> 5) & 31)
+        (index & 31).asInstanceOf[A]
     } else if (xor < (1 << 15)) { // level = 2
-      (display2
-      ((index >>> 10) & 31)
+      display2(
```

Comment:
This left parenthesis is still on the previous line.

Otherwise, looks much better!  Thanks!